projects/plugins/jetpack/extensions/shared/components/product-management-controls/inspector-control.js

```diff
@@ -116,6 +127,20 @@ export default function ProductManagementInspectorControl() {
 								value={ interval }
 							/>
 						</PanelRow>
+						<PanelRow className="donation-subscription">
+							<ToggleControl
+								label={ __( 'Mark subscription as a donation', 'jetpack' ) }
```

Comment:
Let's use the `getMessageByProductType` utility to make sure we are calling the product with the correct type.

E.g.
```js
label={ getMessageByProductType( 'mark this product as a donation', productType ) }
```
...and in `product-management-controls/utils.js`:
```js
const messages = {
	//...
	'mark this product as a donation': {
		[ PRODUCT_TYPE_PAYMENT_PLAN ]: __( 'Mark this payment plan as a donation', 'jetpack' ),
		[ PRODUCT_TYPE_SUBSCRIPTION ]: __( 'Mark this subscription as a donation', 'jetpack' ),
	},
};
```